controllers/volumereplicationgroup_controller.go

```diff
@@ -201,35 +207,78 @@ func (v *VolumeReplicationGroupReconciler) Reconcile(ctx context.Context, req ct
 		return ctrl.Result{}, fmt.Errorf("failed to reconcile VolumeReplicationGroup %w", err)
 	}
 
-	log.Info("Processing VolumeReplicationGroup ", volRepGroup.Spec, " in ns: ", req.NamespacedName)
+	v.ctx = ctx
+
+	if err := v.updatePVCList(); err != nil {
+		log.Error(err, "Handling of Persistent Volume Claims of application failed")
```

Comment:
(nit) maybe add the name of the VRG in the error message. SOmething like
`log.Error(err, "Handling of Persistent Volume Claims of application failed", "name", v.Instance.Name)`

Reply:
Ack!

I was/am not cleaning up logs in this commit. all logs need to move to logr rather than the prometheus log that this part of the code is using. Expect another commit to clean up the logs across this file.